src/rard/research/views/fragment.py

```diff
@@ -988,3 +989,23 @@ def post(self, *args, **kwargs):
                 raise Http404
 
         raise Http404
+
+
+def fetch_fragments(request):
+    # Mention search methods expect a list of search terms so split on space
+    query = request.GET.get("search-fragments")
```

Comment:
I updated this to do two things:
1. make an empty query quickly return none
2. the mention search methods expect a list of search terms, if they're passed a single string, they'll search for each character separately (one of the joys of loose typing)